gtsam/inference/BayesTreeCliqueBase.h

```diff
@@ -166,7 +166,9 @@ namespace gtsam {
     template<class ARCHIVE>
     void serialize(ARCHIVE & ar, const unsigned int /*version*/) {
       ar & BOOST_SERIALIZATION_NVP(conditional_);
-      ar & BOOST_SERIALIZATION_NVP(parent_);
+      if (parent_) { // TODO(fan): Workaround for boost/serialization #119
```

Comment:
That actually does not work, I believe... Also, what is boost issue #119 ? Can you give a link to an *issue* ? Github links to a PR which is not correct I think...

Reply:
https://github.com/boostorg/serialization/issues/119

Reply:
https://github.com/boostorg/serialization/issues/119